docker/diffkemp-devel/Dockerfile

```diff
@@ -67,6 +68,7 @@ RUN apt-get update && \
       python3.8-dev \
       python3.8-distutils \
       python3-setuptools \
+      python-is-python3 \
```

Comment:
I think that I did call wrong python commands inside the container for the whole time.
I am going to check it, because when I am running the container (without my interventions) then it fails due to the missing `pip` executable.

Again, I am going to check it and report back. Either way I am running `pip -e install .` a lot when prototyping, that why I wanted to use it.

Reply:
You can always do `pip3 -e install .`